AirCasting/SearchAndFollow/CompleteScreen/Views/CompleteScreenViewModel.swift

```diff
@@ -141,6 +142,43 @@ class CompleteScreenViewModel: ObservableObject {
         exitRoute()
     }
     
+    private func getAndDisplayData() {
+        if session.stream.first!.sensorName.contains("AirBeam") {
+            do {
+                try streamsDownloader.downloadStreams(with: session.id, for: .airBeam3) { result in
+                    switch result {
+                    case .success(let success):
```

Comment:
Could you consider changing a name of the variable to `downloadedStreams` or something like that instead of `success`? `success` doesn't really say much about what this variable contains